js-library/src/utils/keys.ts

```diff
@@ -0,0 +1,17 @@
+import { utils } from 'ethers'
+
+const { hexStripZeros } = utils
+
+export function isPublicKeyValid(publicKey: string): boolean {
```

Comment:
create a user-defined type guard for the publicKey and for its specific type

Reply:
Also, what does this mean? A function?